src/__tests__/components/profile/EditProfile.test.js

```diff
@@ -1,11 +1,13 @@
 import React from "react";
-import { BrowserRouter, Route } from "react-router-dom";
 import { mount } from "enzyme";
-import { Provider } from "react-redux";
 import store from "../../../store";
+import { BrowserRouter as Router } from "react-router-dom";
+import { Provider } from "react-redux";
 import { EditProfile } from "../../../components/profile/editProfile";
 
 describe("edit profile component tests", () => {
+  let wrapper;
+  let wrapped;
```

Comment:
consider giving better variable names. Instead of `wrapper`, can use `wrapperComponent`, or `wrapperInstance` or any other

Reply:
Thanks @AmosWels ,have renamed these variables.